src/tools/screen.ts

```diff
@@ -59,11 +61,13 @@ const moveMouse = defineTool({
   capability: 'core',
   schema: {
     name: 'browser_screen_move_mouse',
+    title: 'Move mouse',
     description: 'Move mouse to a given position',
     inputSchema: elementSchema.extend({
       x: z.number().describe('X coordinate'),
       y: z.number().describe('Y coordinate'),
     }),
+    type: 'readOnly',
```

Comment:
Any action that generates an input to the user App is potentially destructive. But I guess marking all of such tools destructive is impractical and we follow a "common sense".